opennms-provision/opennms-provision-persistence/src/main/resources/org/opennms/netmgt/provision/persist/default-foreign-source.xml

```diff
@@ -60,6 +60,18 @@
 			<parameter key="type" value="default"/>
 			<parameter key="object" value="org.apache.cassandra.db:type=ColumnFamilies,keyspace=newts,columnfamily=samples"/>
 		</detector>
+		<detector name="JMX-Java8-JVM" class="org.opennms.netmgt.provision.detector.jmx.Jsr160Detector">
+			<parameter key="port" value="18980"/>
+			<parameter key="factory" value="PASSWORD-CLEAR"/>
+			<parameter key="username" value="admin"/>
+			<parameter key="password" value="admin"/>
+			<parameter key="protocol" value="rmi"/>
+			<parameter key="urlPath" value="/jmxrmi"/>
+			<parameter key="timeout" value="3000"/>
+			<parameter key="retries" value="2"/>
+			<parameter key="type" value="default"/>
+			<parameter key="object" value="javax.management.openmbean.SimpleType(name=java.lang.String)"/>
```

Comment:
We can use the VmName attribute to detect the service and use it also for the poller.

<img width="1012" alt="Screenshot 2020-02-24 at 13 51 19" src="https://user-images.githubusercontent.com/1095181/75154093-31357600-570d-11ea-943d-e48df8a36498.png">

Reply:
@indigo423 Take `java.lang:type=Runtime` for the detector. Afaik Runtime and the attributes are only available when a JVM is running.